custom/icds_reports/tests/reports/test_maternal_child_data.py

```diff
@@ -78,17 +79,17 @@ def test_data_stunting_height_for_age(self):
             )['records'][1][0],
             {
                 "redirect": "stunting",
-                "color": "red",
-                "all": 0,
+                "color": "green",
+                "all": 32,
                 "frequency": "month",
                 "format": "percent_and_div",
                 "help_text": "Of the children whose height was measured, the percentage of children between "
                              "0 - 5 years who were moderately/severely stunted in the current month."
                              "<br/><br/>"
                              "Stunting is a sign of chronic undernutrition and has long lasting harmful "
                              "consequences on the growth of a child",
-                "percent": "Data in the previous reporting period was 0",
-                "value": 0,
+                "percent": -14.236111111111107,
+                "value": 19,
```

Comment:
I assume these changes are correct

Reply:
Yeah. this is my fault. I didn't quite get this out in time before these were released. This is the same as https://github.com/dimagi/commcare-hq/pull/21410/files#diff-3e4ca39ec588fce8cc17b5b639188ef6L108 with the exception of percent because the age range was changed between versions